en/telemetry/microhard_serial.md

```diff
@@ -0,0 +1,35 @@
+# Microhard Serial Radio
+
+[Microhard Pico Serial](http://microhardcorp.com/P900.php) radios enable MAVLink communication between a radio on a vehicle and a GCS.  
+Microhard Pico Serial radios are 1 watt output radios that support point to point, point to multi-point, and mesh modes.
+A single ground station radio can be used to communicate with multiple vehicles using point to multi-point or mesh. Vehicles must have different IDs.
+
+![Microhard Radio](../../assets/hardware/telemetry/ark_microhard_serial.jpg)
+
+## Purchase:
+
+* [1W 900MHz Serial Telemetry Radio](https://arkelectron.com/product/1w-900mhz-serial-telemetry-air-radio/)
+* [1W 900MHz USB Serial Telemetry Radio](https://arkelectron.com/product/1w-900mhz-serial-telemetry-ground-radio/)
+* [1W 2.4GHz Serial Telemetry Radio](https://arkelectron.com/product/1w-2400mhz-serial-telemetry-radio/)
+* [1W 2.4GHz USB Serial Telemetry Radio](https://arkelectron.com/product/1w-2400mhz-usb-serial-telemetry-radio/)
+
+### Connecting
+
+Connection on the autopilot is done using a Pixhawk standard 6 pin JST GH Telemetry cable.
+When set to 100mW output power or less, power from the telemtry cable is sufficient.
+For higher radio output powers above 100mW, a battery connection is needed on the 2 Pin Molex Nano-Fit.
+
+Connection on the groundstation is done through USB C. 1W output powers are supported when using USB PD.
+
+## Setup/Configuration
+
+Using [Pico Config](https://arkelectron.com/wp-content/uploads/2020/09/PicoConfig-1.7.zip) on Windows, the radio settings can be adjusted.
```

Comment:
Is this only configurable on windows?

Reply:
PicoConfig only works on windows. If there is enough interest in the future, I could make a command line version for linux. The radios can also be configured through the main serial port using modem AT commands.